### PR TITLE
Add overcloud image DIB config

### PR DESCRIPTION
**Pull request by cityofships**
Initial commit of what's targeted as a _standard_ CentOS 8 Stream deployment image.

### CODE REVIEW

File: etc/kayobe/overcloud-dib.yml [hunk 1]
```diff
@@ -0,0 +1,157 @@
+---
```
Comment (markgoddard) [inline etc/kayobe/overcloud-dib.yml:1]: Missing comments

**Review by markgoddard — COMMENTED** [review]
Has this come from SOS?

I think we are targeting this for Xena.

**Comment by markgoddard** [comment]
Ticket for this: https://stackhpc.atlassian.net/browse/DEV-852

**Comment by mnasiadka** [comment]
> Has this come from SOS?
> 
> I think we are targeting this for Xena.

What is SOS?

**Comment by markgoddard** [comment]
> > Has this come from SOS?
> > I think we are targeting this for Xena.
> 
> What is SOS?

Scientific OpenStack!

**Comment by mnasiadka** [comment]
> > > Has this come from SOS?
> > > I think we are targeting this for Xena.
> > 
> > 
> > What is SOS?
> 
> Scientific OpenStack!

I only know it for emergency call sign, and it sounded weird!

**Comment by oneswig** [comment]
A swap partition would be good

**Comment by mnasiadka** [comment]
> 

I think we usually add this post-deployment, easier to tune the size per configuration (or not have it at all - like on Ceph nodes).

**Comment by markgoddard** [comment]
I'm hoping to iterate on this as part of the RUG deployment.

**Comment by markgoddard** [comment]
> I'm hoping to iterate on this as part of the RUG deployment.

https://github.com/stackhpc/stackhpc-kayobe-config/pull/99

**Pull request closed without merging**